.github/workflows/reusable-build-flavor.yaml

```diff
@@ -135,3 +135,16 @@ jobs:
         if: ${{ inputs.model == 'generic' }}
         run: |
           docker push ${{ env.IMAGE_NAME }}
+      - name: Tag and Push Nightly
+        if: github.ref == 'refs/heads/master'
```

Comment:
do we need to check if the action is a push or just nothing else would have that ref?

Reply:
good question, I think this is enough but not 100% ... maybe i test and see what happens